explore/preferences.md

```diff
@@ -10,7 +10,7 @@ redirect_from:
 
 Storing user configurations and values is a common task for application developers, but implementing it across multiple platforms can be tedious and time-consuming. To make it easier, Fyne has an API for storing values on the filesystem in a clean and understandable way while the complex parts are handled for you.
 
-Lets start with the setup of the API. It is part of the [Preferences](https://pkg.go.dev/fyne.io/fyne/v2?tab=doc#Preferences) interface where storage and loading functions exist for values of Bool, Float, Int and String. They each consist of three different functions, one for loading, one loading with a fallback value and lastly, one for storing values. An example of the three functions and their behaviour can be seen below for the String type:
+Lets start with the setup of the API. It is part of the [Preferences](https://pkg.go.dev/fyne.io/fyne/v2?tab=doc#Preferences) interface where storage and loading functions exist for values of `bool`, `[]bool`, `float64`, `[]float64` `int`, `[]int`, `string` and `[]string`. They each consist of three different functions, one for loading, one loading with a fallback value and lastly, one for storing values. An example of the three functions and their behaviour can be seen below for the `string` type:
```

Comment:
to make this change appropriate I think you would need to add the word `type` at the end of the type list.
I would also consider not listing them all directly but listing the primitives then something like "and lists of each as well" (or slice of depending on how you word it).

Reply:
Hope now is better. I have also made some other minor changes inside the same file.